internal/handler/v1/router.go

```diff
@@ -8,17 +8,53 @@ import (
 )
 
 func RegisterRoutes(r *router.Router) {
-	AddCustomRoutes(r)
-	AddProxyRoutes(r)
-	AddPriceRoutes(r)
-	AddERC20Routes(r)
-	AddUtilsRoutes(r)
-	AddNetworkRoutes(r)
-	AddValidatorDirectoryRoutes(r)
-	AddAirtableRoutes(r)
-	AddTransactionRoutes(r)
-	AddBalancesRoutes(r)
-	AddStakingRoutes(r)
-	AddGovernanceRoutes(r)
-	AddValidatorsRoutes(r)
+	// epoch
+	r.GET("/RemainingEpochs", RemainingEpochs)
+	r.GET("/Epochs/{chain}", Epochs)
+
+	// announcements
+	r.GET("/Announcements", GetAnnouncements)
+
+	// config
+	r.GET("/NetworkConfig", NetworkConfig)
+	r.GET("/NetworkConfig/{name}", NetworkConfigByName)
+
+	// ibc
+	r.POST("/ibcTransfer", IBCTransfer)
+
+	// bank
+	r.GET("/BalanceByDenom/{chain}/{address}/{denom:*}", BalanceByDenom)
+	r.GET("/BalanceByNetworkAndDenom/{chain}/{token}/{address}", BalanceByNetworkAndDenom)
+	r.GET("/EVMOSIBCBalance/{chain}/{address}", EVMOSIBCBalance)
+
+	// distribution
+	r.POST("/rewards", Rewards)
```

Comment:
is this the same as withdraw in the notion page ?

Reply:
correct!